backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/getBPOpenAmtToDate.sql

```diff
@@ -6,7 +6,7 @@ CREATE OR REPLACE FUNCTION getBPOpenAmtToDate(p_AD_Client_ID numeric,
                                               p_C_BPartner_id numeric,
                                               p_C_Currency_ID numeric,
                                               p_UseDateAcct text = 'Y',
-                                              p_IsSOTrx text = 'Y')
+                                              p_IsSOTrx text = NULL)
```

Comment:
IMHO this shall stay mandatory.
Not sure if we can legally mix vendor side with customer side,
so let's keep it as it is if u don't have explicit requirements about it.
nevertheless, pls MAKE SURE you are documenting this in our tests.

cc @cristinamghita

Reply:
also, in BusinessPartnerAccountSheetReport, when you call getBPOpenAmtToDate,
pls make sure you are calling it with t.IsSOTrx and not 'Y'.